samples/bluetooth/peripheral/prj.conf

```diff
@@ -16,10 +16,11 @@ CONFIG_BT_DEVICE_APPEARANCE=833
 CONFIG_BT_DEVICE_NAME_DYNAMIC=y
 CONFIG_BT_DEVICE_NAME_MAX=65
 
-CONFIG_BT_KEYS_OVERWRITE_OLDEST=y
-CONFIG_BT_SETTINGS=y
-CONFIG_FLASH=y
-CONFIG_FLASH_PAGE_LAYOUT=y
-CONFIG_FLASH_MAP=y
-CONFIG_NVS=y
-CONFIG_SETTINGS=y
+# CONFIG_BT_KEYS_OVERWRITE_OLDEST=y
```

Comment:
The changes to this file seem unrelated to the rest of the commit.  Are they intended?